### PR TITLE
bump version and create release note stubs for 1.1.0

### PR DESCRIPTION
**Pull request by grlee77**

### CODE REVIEW

File: doc/release/1.1.0-notes.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
+==============================
+PyWavelets 1.0.0 Release Notes
```
Comment (rgommers) [inline doc/release/1.1.0-notes.rst]: 1.1.0

File: doc/release/1.1.0-notes.rst [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+==============================
+
+.. contents::
+
+We are very pleased to announce the release of PyWavelets 1.1.
+
+This release drops requires Python >=3.5 and has dropped Python 2.7 support.
```
Comment (rgommers) [inline doc/release/1.1.0-notes.rst]: " drops requires" --> "requires"

File: doc/release/1.1.0-notes.rst [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+Other changes
+=============
+
+Authors
+=======
+
+Issues closed for v1.1.0
+------------------------
+
+Pull requests for v1.0.0
```
Comment (rgommers) [inline doc/release/1.1.0-notes.rst]: 1.1.0

**Comment by grlee77** [comment]
Thanks. It looks like I needed to do more proofreading!

**Pull request merged**